README.rst

```diff
@@ -1,7 +1,7 @@
 Snowflake SQLAlchemy
 ********************************************************************************
 
-.. image:: https://codecov.io/gh/snowflakedb/snowflake-sqlalchemy/branch/master/graph/badge.svg
+.. image:: https://codecov.io/gh/snowflakedb/snowflake-sqlalchemy/branch/main/graph/badge.svg?token=1kmCWB3WRS
```

Comment:
should this token be included? just want to make sure no credential is leaked

Reply:
I hope it's not sensitive, the whole `.rst` copy-paste is generated by codecov